### PR TITLE
fix: vitamin start with env=production

### PR DESCRIPTION
**Pull request by alepee**
when starting vitamin in production environment, vitamin server was never called.
with @johangirod

### CODE REVIEW

File: bin/vitamin.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const createCompiler = (webpackConfig, message, options) => {
     const compiler = webpack(webpackConfig);
     if (process.stdout.isTTY) {
         const bar = new ProgressBar(
-            `${chalk.blue(`\uD83D\uDD50 Building ${message}...`)} :percent [:bar]`,
+            `${chalk.blue(`\uD83D\uDD50  Building ${message}...`)} :percent [:bar]`,
```
Comment (christophehurpeau) [inline bin/vitamin.js]: why did you add a space?

Comment (johangirod) [inline bin/vitamin.js]: Because it was adjacent to the UTF8 symbol before.I think it's because the clock symbol takes two character spaces but I'm not sure.

Comment (christophehurpeau) [inline bin/vitamin.js]: perhaps create a emoj object: `{ clock: '\uD83D\uDD50 ' }` => `${chalk.blue(`${emoj.clock} Building ${message}...`)} :percent [:bar]`
